data/hai/hai reveal 4 middle.txt

```diff
@@ -1412,33 +1412,180 @@ mission "Hai Reveal [A11] Sound the Alarm"
+			`	The meeting's room is shaped like a wok, with the Elders towering over a circle in the center of witnesses and a moderator. A wedge of the room is filled with holoprojectors, machinery, entrances, and walkways. Ever-circling holographic charts and graphs display indecipherable information. There are hovering bricks of cameras moving around the room, presumably for broadcasting to Hai media, or recording for posterity. One of them seems fixated on you as you find your assigned seat between Sayari and Teeneep.`
+			`	After everyone is set in place, the moderator abruptly starts. "Teeneep has provided a confession about her visit to the Unfettered, dangerous media activities, disruptions in human space, and other offenses. In this live broadcast, the Elder Council will decide whether to censure or exile Teeneep. All relevant evidence will be available in the video feed's metadata. Two Elders have questions for Captain <last> before the Council votes."`
+			`	At the word 'confession,' Teeneep is offended and Sayari is worried.`
+			choice
+				`	"Sayari, what is happening?"`
+				`	(Remain silent.)`
+					goto question1
+			`	Sayari explains. "The Elder Council has always been concerned by Teeneep's activism. They're especially fearful of her outreach to the Unfettered and humans. It seems they've finally had their fill and decided she's a threat to the peace. In this trial, Elders will decide will either ban her from public discourse, or exile her from most of Hai space. If she's lucky, they'll do neither."`
+			`	"Ah, but it's broadcast live," reassures Teeneep. "Like any oligarchy, the Elders normally hide their biases and misdeeds behind a veil of secrecy. Even if they take me down, the movement I'm part of will use this broadcast to bring down the current ruling Council.`
+			`	Teeneep gives you a stern warning. "Be careful what you say in this public broadcast, Captain <last>. Divulging classified information may get you in trouble here or at home. It could cause widespread panic that would negate any good we might have done. Speak with generalities, but make your point abundantly clear."`
+			label question1
+			# FIXME: Need a gender-neutral synonym for "errand boy" that has the same connotations.
+			`	The Elder that was dismissive of Teeneep at her last meeting has your first question. "I understand Teeneep has tricked you into being her errand boy. You have my sympathy. During her misguidance, have you seen any evidence of validity of her conspiracy theories? Are the pirate attacks part of a coordinated effort?"`
```

Comment:
```suggestion
			`	The Elder that was dismissive of Teeneep at her last meeting has your first question. "I understand Teeneep has tricked you into being her personal peon. You have my sympathy. During her misguidance, have you seen any evidence of validity of her conspiracy theories? Are the pirate attacks part of a coordinated effort?"`
```
or "errand runner", or "errand kid"?  Peon might make people think differently.

Reply:
What about "gopher?" It has a similar meaning, and would be somewhat humorous coming from a squirrel.

Reply:
Sure!

Reply:
Teeneep has now "enlisted you as her gopher."